databases/metabase/Portfile

```diff
@@ -21,12 +21,13 @@ supported_archs     noarch
 license             MIT
 
 master_sites        https://downloads.metabase.com/v${version}/
+dist_subdir         ${name}/${version}
```

Comment:
why is this needed; does the default not work?

Reply:
The distfiles name is metabase.jar and there is no version name, when installing new version with the port install -b option, the old version matebase.tbz2 on the macports will cause the checksum failed.

This issue came up in the jenkins port, fixed the same way in this port.

ref -> https://github.com/macports/macports-ports/pull/20110#issuecomment-1692766631